RockLib.Diagnostics/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 3.0.1 - 2024-07-15
+
+### Changed
+RockLib.Configuration.4.0.0 -> RockLib.Configuration.4.0.1
+System.Text.Json.8.0.0 -> System.Text.Json.8.0.4
```

Comment:
Did you actually change this version directly, or is this coming from RockLib.Configuration?

Reply:
it's coming from RockLib.Configuration